.kitchen.yml

```diff
@@ -3,7 +3,7 @@
 driver:
   name: vagrant
   network:
-    - ["private_network", {ip: "192.168.35.35"}]
+    - ["private_network", {ip: "192.168.35.36"}]
```

Comment:
Why this change?

Reply:
Changed this so more than one project can be tested at a time. I had two projects using the same ip address and tests failed. Bumped this one to 36.

Reply:
Two projects on the same build agent? If we run into collisions we should start looking into dynamic allocation or else this may get unwieldy.